pkgs/development/python-modules/bbox/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pyquaternion
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "bbox";
+  version = "0.9.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ucR7mg9eubEefjC7ratEgrb9h++a26z8KV38n3N2kcw=";
+  };
+
+  pythonImportsCheck = [ "bbox" ];
+
+  propagatedBuildInputs = [ pyquaternion numpy ];
```

```suggestion
  propagatedBuildInputs = [ pyquaternion numpy ];

   pythonImportsCheck = [ "bbox" ];
```

Be more inlined with when they are relevant.